flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/stream/StreamingSink.java

```diff
@@ -173,7 +173,7 @@ public static DataStreamSink<?> sink(
         }
 
         DataStreamSink<?> discardingSink =
-                stream.addSink(new DiscardingSink<>()).name("end").setParallelism(1);
+                stream.sinkTo(new DiscardingSink<>()).name("end").setParallelism(1);
```

Comment:
I'm not sure if this will cause compatibility issues, at least JsonPlan should be different.

Reply:
Do you mean `json_execution_plan` or `execNode plan` described in[ FLIP-190 ](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=191336489#FLIP190:SupportVersionUpgradesforTableAPI&SQLPrograms-GeneralJSONPlanAssumptions )
As long as we can keep `execNode plan` same, there won't break `compatibility`, And as i can see, the `execNode plan`  won't change by this pr, so I don't think it'll break 
 `compatibility`

Reply:
Yes, I means change the json format `== Physical Execution Plan ==`. For example, changing the type of an operator.

Reply:
If it's only Physical Execution Plan, I don't think it will break compatibility.

Reply:
Thank you the confirmation, make sense to me.